impeller/geometry/matrix_unittests.cc

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "gtest/gtest.h"
+
+#include "flutter/impeller/geometry/matrix.h"
+
+#include "flutter/impeller/geometry/geometry_asserts.h"
+
+namespace impeller {
+namespace testing {
+
+TEST(MatrixTest, Multiply) {
```

Comment:
Having this test as documentation on how to transform a quad would have saved me a few minutes.